api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -208,3 +226,71 @@ message DeleteUnavailabilityWindowRequest {
 message DeleteUnavailabilityWindowResponse {
   // Currently no payload in the response.
 }
+
+// A request for the `OpenGroundStationStream` method.
+message GroundStationStreamRequest {
+  // The ID of the ground station that is opening a stream. The ID of a ground station can be found
+  // on the StellarStation Console page for the ground station.
+  string ground_station_id = 1;
+
+  // A request to send on the ground station stream. The request does not need to be filled on the
+  // first request to `OpenGroundStationStream`.
+  oneof Request {
+    SendTelemetryRequest send_telemetry_request = 2;
+
+    // An event that occurred while processing the stream. Satellite operators will often use these
+    // events for debugging and monitoring.
+    //
+    // - `command_sent` must be returned right after completing transmission of
+    //    commands received in a `SendSatelliteCommandsResponse`.
+    // - `plan_monitoring_event.ground_station_configuration` must be returned once at the beginning
+    //   of the execution of a plan and contains information about the actual configuration of the
+    //   ground station at the time.
+    // - `plan_monitoring_event.ground_station_state` must be sent continuously throughout the
+    //   plan and contains real-time monitoring of the hardware of the ground station.
+    // - `plan_monitoring_event.ground_station_event` must be sent once at each of the transitions
+    //   documented in `PlanLifecycleEvent` when executing a plan.
+    StreamEvent stream_event = 3;
+  }
+}
+
+// A request to send telemetry received from a satellite to a user.
+message SendTelemetryRequest {
+  // The telemetry being sent.
+  Telemetry telemetry = 1;
+
+  // The ID of the plan the telemetry is being sent for.
+  string plan_id = 2;
```

Comment:
I know this is for to keep symmetry with the satellite stream but I guess it would be OK to reverse these.